src/plugins/condition/components/Condition.vue

```diff
@@ -229,7 +230,29 @@ export default {
     },
     methods: {
         initialize() {
-            this.setOutput();
+            this.setOutputSelection();
+            if (!this.domainObject.isDefault) {
+                this.conditionClass = new ConditionClass(this.domainObject, this.openmct);
```

Comment:
This code was removed in topic-conditionals. Can you double check that your code is upto date with topic-conditionals and you didn't accidentally add some stuff back?

Reply:
done